lib/pdk/tests/unit.rb

```diff
@@ -89,7 +89,7 @@ def self.invoke(report, options = {})
 
         result[:exit_code]
       ensure
-        tear_down
+        tear_down if options[:'clean-fixtures']
```

Comment:
Users forgetting tear-down and then running `pdk build` would end up bundling their fixtures. Until `pdk release prep` can clean the fixtures, I would suggest having `pdk build` run a `spec_clean` or `tear_down` or whatever to avoid bundling fixtures.

Reply:
@hunner good call!

Reply:
@rodjek has this been ticketed so we don't forget to update pdk build? I'm wondering if we should add the fixtures directory to the .pdkignore vs calling something that will change a tagged repo/directory before building.